src/node/desktop/src/main/session-launcher.ts

```diff
@@ -191,4 +193,34 @@ export class SessionLauncher {
   closeAllSatellites(): void {
     console.log('CloseAllSatellites not implemented');
   }
+
+  getLauncherToken(): string {
+    if (!this.launcherToken) {
```

Comment:
Do you think there's an appropriate place where we could generate this on initialization, rather than just generating on first use (and re-checking each time thereafter)?

Reply:
I was just following the existing code. Agree it's a bit odd.